Jenkinsfile

```diff
@@ -0,0 +1,38 @@
+@Library('xmos_jenkins_shared_library@v0.33.0') _
+
+getApproval()
+
+pipeline {
+  agent {
+    label 'x86_64 && linux'
+  }
+  environment {
+    XTC_VERSION = '15.3.0'
+  }
+  options {
+    buildDiscarder(xmosDiscardBuildSettings())
+    skipDefaultCheckout()
+    timestamps()
+  }
+  stages {
+    stage('Build') {
+      steps {
+        dir("lib_random") {
+          checkout scm
```

Comment:
Lib checks missing?

Reply:
There weren't any library checks in this repository, so they aren't being added at this point.